pkgs/tools/nix/dnadd/default.nix

```diff
@@ -0,0 +1,23 @@
+{stdenv, fetchFromGitHub, gcc}:
+
+stdenv.mkDerivation rec {
+  pname = "dnadd";
+  version = "1.0.0";
+  ownerj="JoeLancaster";
+  src = fetchFromGitHub {
+    owner = ownerj;
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1ljskfhvc68rw4w5vwknrddknh1xk3iw1z635am7si3x0k9m4ngr";
+  };
+  installPhase = ''
+                 mkdir -p $out
+                 make DESTDIR=$out install
```

Comment:
I would suggest using `PREFIX` instead of `DESTDIR` to better match the [GNU standards](https://www.gnu.org/prep/standards/html_node/Directory-Variables.html#index-libexecdir). `DESTDIR` has a [different meaning](https://www.gnu.org/prep/standards/html_node/DESTDIR.html#DESTDIR) and should not be used in Nixpkgs (https://github.com/NixOS/nixpkgs/issues/65718).